### PR TITLE
kelly/fix

### PR DESCRIPTION
**Pull request by kellybinary**
- update symbol in `trade_store` when selecting an active position in position drawer
- add `getQueryString` method in `url-helper.js`
- store query string in `trade_store` properly
- refactor allow-equals logic from `trade_store` to it's own component

### CODE REVIEW

File: src/javascript/app_2/Utils/URL/url-helper.js [hunk 1]
```diff
@@ -32,12 +31,13 @@ export default class URLHelper {
 
             const value = params[name];
 
+            // console.log(name, value);
```
Comment (OskarAhl) [inline src/javascript/app_2/Utils/URL/url-helper.js]: Think this can be removed

File: src/javascript/app_2/Utils/URL/url-helper.js [hunk 2]
```diff
@@ -10,8 +10,7 @@ export default class URLHelper {
      */
     static getQueryParams(url) {
         const query_string =  url ? new URL(url).search : window.location.search;
-        const query_encoded = encodeURIComponent(query_string);
```
Comment (OskarAhl) [inline src/javascript/app_2/Utils/URL/url-helper.js]: What was the reason for the encode again? @cakasuma

File: src/javascript/app_2/Stores/Modules/Trading/trade-store.js [hunk 3]
```diff
@@ -97,6 +97,9 @@ export default class TradeStore extends BaseStore {
     @observable proposal_info        = {};
     @observable purchase_info        = {};
 
+    // Query string
+    @observable query = '';
```
Comment (OskarAhl) [inline src/javascript/app_2/Stores/Modules/Trading/trade-store.js]: What's the reason for adding the query as an observable?

File: src/javascript/app_2/Utils/URL/url-helper.js (hunk 2 above)
Comment (kellybinary) [reply src/javascript/app_2/Utils/URL/url-helper.js]: @OskarAhl @cakasuma thanks, i've added a fix

**Pull request merged**